src/q8/paradigms-SINF2335/exam/2016/Juin/All/paradigms-SINF2335-exam-2016-Juin-All.tex

```diff
@@ -19,34 +19,27 @@ \section{Define the following concepts in your own words, and give
 taken from your own experience.}
 
 \begin{description}
-    \item[reification]: making the ​language concepts available for manipulation
-    by the programmer.
+    \item[reification]: making the language concepts available for manipulation by the programmer.
         \subitem Example: in Ruby, the class \verb#Class# represents a class.
-        \subitem Exemple: La classe java.lang.Class $\rightarrow$ newInstance(),
+        \subitem Example: La classe java.lang.Class $\rightarrow$ newInstance(),
```

Comment:
Everything should be in english here:

`Example: the \lstinline[language=java]|java.lang.Class| $\rightarrow$ \lstinline[language=java]|newInstance()| `